docs/whitepapers/kalmanfilter/kalmanfilter.tex

```diff
@@ -0,0 +1,345 @@
+
+\begin{equation}
+\dot{z}_t = a^z_{T_t} - g
+\end{equation}
+
+\noindent
+Here, \( a^z_{T_t} \) signifies the upward acceleration generated by the propulsion at time \( t \), measured in meters per second squared. The constant \( g \) denotes the acceleration due to Earth's gravity, also in meters per second squared. The altitude \( z_t \) represents the drone's center of mass's vertical position at time \( t \), measured in meters.
+\\
+
+\noindent
+Additionally, the drone's vertical velocity \( v_{z_t} \) and vertical acceleration \( a_{z_t} \) are defined by the rate of altitude change \( \dot{z}_t \) and the rate of vertical acceleration change \( \dot{a}^z_{T_t} - g \), respectively. The term \( \dot{a}^z_{T_t} \) is derived from the quadcopter's upward thrust and serves as the system's input, while \( g \) is considered a constant input in the opposite direction.
+\\
+
+\noindent
+Let \( y_{t}^z = z_t \) be the output equation of the system. Then the corresponding state-space representation is,
+
+\begin{equation}
+\begin{bmatrix}
+v_{v,t}\\
+a_{t}^z
+\end{bmatrix} =
+\begin{bmatrix}
+0 & 1 \\
+0 & 0 
+\end{bmatrix}
+\begin{bmatrix}
+z\\
+v_{v,t}
+\end{bmatrix} + 
+\begin{bmatrix}
+0 & 0 \\
+1 & 1 \\ 
+\end{bmatrix}
+\begin{bmatrix}
+a_{T,t}^z \\ 
+-g
+\end{bmatrix},
+\end{equation}
+\\
+
+\noindent
+for the Barometer sensor's output, denoted by \( y_{barom} \), is described by the equation:
+\begin{equation}
+y^{barom} = z + bias + V^{barom}
+\end{equation}
+\\
+where, 
+\begin{equation}
+bias_{t+1} = bias_t + W_{t}^{bias}
+\end{equation}
+the bias should stay consistent throughout readings 
+\\
+\\ 
+
+
+\noindent
+for the GPS sensor's output, denoted by \( y_{gps} \), is described by the equation:
+\begin{equation}
+y_{gps} = z + V^{gps}
+\end{equation}
+
+\begin{equation}
+y_{gps} =
+\begin{bmatrix}
+    1 & 0
+\end{bmatrix}
+x + V^{gps}
+\end{equation}
+\\
+
+\noindent
+The Time-of-Flight (ToF) sensor's output, denoted by \( y_{ToF} \), is described by the equation:
+
+\begin{equation}
+y_{ToF} = z + V^{ToF}
+\end{equation}
+\\
+
+\noindent
+throught all sensors \( y \) represents the output from the sensor. The variable \( z \) signifies the quadcopter's altitude, which is the measurement for all sensors. The term \( V \) encapsulates the measurement noise or errors associated with the sensors. This noise term, \( V \), encompasses various factors such as sensor inaccuracies, the impact of environmental conditions on sensor performance, and any systematic bias that might be inherent in the sensor's readings.
+\\
+
+\noindent
+Equation (2) describes the continuous-time altitude dynamics of the quadcopter. The dis-
+cretization of the altitude dynamics of the system (2) with a sampling frequency of Ts using the zero-order hold technique is,
```

Comment:
Use `\eqref{}` and `\label{}`

Reply:
can you show me how in the meeting later? having some trouble with this

Reply:
### Example

```latex
Here is an equation 
\begin{equation}
F = m\ddot{x}.\label{eq:newton-second-law}
\end{equation}
Additionally, we know that 
\begin{align}
x {}={}& \sqrt{y^2 + 1},
\\
z {}={}& a + 1,
\end{align}
therefore, by virtue of Equation \eqref{eq:newton-second-law},
blah blah.

This is a new paragraph.
```